src/Rpc/NodeGuardService.cs

```diff
@@ -868,6 +870,28 @@ private bool ValidateTargetBalance(LiquidityRule rule)
         return true;
     }
 
+    public override async Task<GetUtxosResponse> GetUtxos(GetUtxosRequest request, ServerCallContext context)
+    {
+        var wallets = await _walletRepository.GetAvailableWallets();
+        List<Utxo> utxos = new();
+        foreach (var wallet in wallets)
+        {
+            var walletUtxos = await _fmutxoRepository.GetByWalletId(wallet.Id);
+            utxos.AddRange(walletUtxos.Select(utxo => new Utxo()
+            {
+                Amount = new Money(utxo.SatsAmount),
+                // We follow the same convention as GetAvailableUtxos for displaying outpoints
+                Outpoint = $"{utxo.TxId}-{utxo.OutputIndex}",
```

Comment:
you already have this in the model!
```
        public override string ToString()
        {
            return $"{TxId}:{OutputIndex}";
        }
```

Reply:
we follow the convention of the nbxplorer, so we have to do it with a dash instead of the colon